README.md

```diff
@@ -22,8 +22,8 @@ Then, you need to add some informations in your configuration files
 ```php
 'providers' => [
     ...
-    Mailjet\LaravelMailjet\MailjetServiceProvider::class,
-    Mailjet\LaravelMailjet\MailjetMailServiceProvider::class,
+    Mailjet\LaravelMailjet\Providers\MailjetServiceProvider::class,
```

Comment:
Agreed, by the way I have for the moment left the Facade, for MailjetServiceProvider and implemented a separate (new) contract + service provider. Do you think I should just leave MailjetServiceProvider in the old namespace, to avoid breaking changes ?